Ginger/Ginger/MainWindow.xaml.cs

```diff
@@ -106,7 +106,9 @@ public void Init()
                     Reporter.ToStatus(eStatusMsgKey.ExitMode);
                 }
                  WorkSpace.Instance.UserProfile.GingerStatus = eGingerStatus.Active;
+                Console.WriteLine("Before Save Profile");
                  WorkSpace.Instance.UserProfile.SaveUserProfile();
+                Console.WriteLine("After Save profile");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)